spray-http/src/test/scala/spray/http/UriSpec.scala

```diff
@@ -358,8 +366,12 @@ class UriSpec extends Specification {
       normalize("Http://Localhost") === "http://localhost"
       normalize("hTtP://localHost") === "http://localhost"
       normalize("https://:443") === "https://"
+      normalize("https://:444") === "https://:444"
+      normalize("http://:80/foo") === "http:///foo"
+      normalize("http://:8080/foo") === "http://:8080/foo"
       normalize("ftp://example.com:21") === "ftp://example.com"
-      normalize("example.com:21") === "example.com:21"
+      normalize("example.com:21") === "example.com:21" // example.com is parsed as the SCHEME (which is correct)
```

Comment:
In Akka, this test expects "example.com:21" to be normalized to example.com://21 (and the corresponding PR changes this line accordingly). I guess this case is analogous to the "g:h"-case so no slashes should appear here as well.